cf_units/__init__.py

```diff
@@ -220,10 +220,15 @@
 # load the libudunits2 shared library
 #
 if _lib_ud is None:
-    _lib_ud = config.get_option(
+    udunits2_c = config.get_option(
```

Comment:
The convention so far has been for these kinds of helper variables to have an underscore prefix to mark them as private and stop them from turning up in tab-completion.